funnel/templates/event.html

```diff
@@ -76,6 +76,37 @@
 {% endblock %}
 {% block footerscripts %}
 <script>
+  if (Modernizr.localstorage) {
+    LocalStorageKey = 'p-id';
+
+    Storage.prototype.set = function(key, value) {
+      var participantList = JSON.parse(this.getItem(LocalStorageKey));
+      if(participantList) {
+        participantList[key] = value;
+      }
+      else {
+        participantList = {};
+        participantList[key] = value;
+      }
+      return this.setItem(LocalStorageKey, JSON.stringify(participantList))
+    }
+    Storage.prototype.get = function(key) {
+      var participantList = JSON.parse(this.getItem(LocalStorageKey));
+      return participantList[key];
+    }
+    Storage.prototype.getItems = function() {
+      var participantList = JSON.parse(this.getItem(LocalStorageKey));
+      return participantList;
+    }
+    Storage.prototype.remove = function(key) {
+      var participantList = JSON.parse(this.getItem(LocalStorageKey));
+      if(participantList) {
+        delete participantList[key];
+      }
+      return this.setItem(LocalStorageKey, JSON.stringify(participantList))
+    }
+  }
+
```

Comment:
Incase of no localStorage, it falls back to earlier method of reloading the page after updating DB